### PR TITLE
MB-12542 Part 2: Two column component for shipment evaluation reports

### PR DESCRIPTION
**Pull request by Ryan-Koch**
## Summary

Adds a two column style shipment display component for the shipment counseling report page.

## Setup to Run Your Code

<details>
<summary>💻 You will need to use three separate terminals to test this locally.</summary>

##### Terminal 1

Start the Storybook locally.

```sh
make storybook
```

##### Terminal 2

Start the UI locally.

```sh
make client_run
```

##### Terminal 3

Start the Go server locally.

```sh
make server_run
```

</details>

### Additional steps

1. Access the office app
2. Login as an office user
3. Navigate to a move and to the quality assurance tab
4. Click create report or view report under shipment evaluation reports
5. Check out the fancy new thing

## Verification Steps for Author

These are to be checked by the author.

- [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
- [ ] Request review from a member of a different team.
- [ ] Have the Jira acceptance criteria been met for this change?

## Verification Steps for Reviewers

These are to be checked by a reviewer.

### Frontend

- [ ] User facing changes have been reviewed by design.
- [ ] There are no aXe warnings for UI.
- [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/master/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
- [ ] There are no new console errors in the browser devtools
- [ ] There are no new console errors in the test output
- [ ] If this PR adds a new component to Storybook, it ensures the component is fully responsive, OR if it is intentionally not, a wrapping div using the `officeApp` class or custom `min-width` styling is used to hide any states the would not be visible to the user.

### Backend

- [ ] Code follows the guidelines for [Logging](https://transcom.github.io/mymove-docs/docs/dev/contributing/backend/Backend-Programming-Guide#logging)
- [ ] The requirements listed in [Querying the Database Safely](https://transcom.github.io/mymove-docs/docs/dev/contributing/backend/Backend-Programming-Guide/#querying-the-database-safely) have been satisfied.

#### Any new migrations/schema changes:

- [ ] Follows our guidelines for [Zero-Downtime Deploys](https://transcom.github.io/mymove-docs/docs/dev/contributing/database/Database-Migrations#zero-downtime-migrations)
- [ ] Have been communicated to #g-database
- [ ] Secure migrations have been tested following the instructions in our [docs](https://transcom.github.io/mymove-docs/docs/dev/contributing/database/Database-Migrations#secure-migrations)

## Screenshots
image
image
image
image

### CODE REVIEW

**Review by cosburn — APPROVED** [review]
Nice!

File: src/components/Office/DefinitionLists/ShipmentInfoList.jsx [hunk 1]
```diff
@@ -166,6 +228,50 @@ const ShipmentInfoList = ({ className, shipment, warnIfMissing, errorIfMissing,
...
+              styles.tableDisplay,
+              styles.compact,
+              className,
+            )}
+            data-testid="shipment-info-list-left"
+          >
+            {showElement(scheduledPickupDateElement) && scheduledPickupDateElement}
+            {showElement(actualPickupDateElementFlags) && actualPickupDateElement}
+            {showElement(requestedDeliveryDateElementFlags) && requestedDeliveryDateElement}
+            {showElement(agentsElementFlags) && releasingAgentElement}
```
Comment (mboren) [inline src/components/Office/DefinitionLists/ShipmentInfoList.jsx:249]: in the design, it looks like requested pickup date comes before actual pickup date

File: src/components/Office/DefinitionLists/NTSRShipmentInfoList.jsx [hunk 2]
```diff
@@ -188,6 +277,48 @@ const NTSRShipmentInfoList = ({
...
+              styles.descriptionList,
+              styles.tableDisplay,
+              styles.compact,
+              className,
+            )}
+            data-testid="shipment-info-list"
+          >
+            {isExpanded && scheduledPickupDateElement}
+            {isExpanded && actualPickupDateElement}
+            {isExpanded && storageFacilityContactInfoElement}
```
Comment (mboren) [inline src/components/Office/DefinitionLists/NTSRShipmentInfoList.jsx:297]: based on the design, it looks like requested pickup date might be missing from this column
image

**Review by mboren — APPROVED** [review]
Looks good.
I noted a couple of places where I noticed minor differences in which fields are where from the design. I think it would be OK to address those in followup work if they don't fit in to the current scramble. There's also a couple of `!important`s in the CSS, but it sounds like you're planning to tinker with the CSS some more in a followup story.

File: src/components/Office/DefinitionLists/ShipmentInfoList.jsx (hunk 1 above)
Comment (Ryan-Koch) [reply src/components/Office/DefinitionLists/ShipmentInfoList.jsx:249]: Ah yeah that was a change we made to the design this morning. I'm going to fix that in the next story 👍

File: src/components/Office/DefinitionLists/NTSRShipmentInfoList.jsx (hunk 2 above)
Comment (Ryan-Koch) [reply src/components/Office/DefinitionLists/NTSRShipmentInfoList.jsx:297]: Same thing as above for this one.

**Pull request merged**